skaffold.yaml

```diff
@@ -4,14 +4,12 @@ kind: Config
 build:
   insecureRegistries:
     - registry.example.com
-  tagPolicy:
-    gitCommit: {}
+  local:
+    push: true
   artifacts:
     - image: registry.example.com/kanopy-gateway-cert-controller
-      kaniko:
-        skipTLS: true
-        buildArgs:
-  cluster: {}
+      ko:
```

Comment:
kaniko build was failing due to https://github.com/GoogleContainerTools/kaniko/issues/2164

I've changed this to align with https://github.com/kanopy-platform/argoslower/blob/main/skaffold.yaml, which works